src/pages/verify/e-ids/dutch-idin.mdx

```diff
@@ -0,0 +1,239 @@
+---
+product: verify
+category: eIDs
+sort: 10
+title: Dutch iDIN
+subtitle: Learn more about Dutch iDIN token contents available in the four different user profiles.
+---
+
+import Layout from '../../../layouts/mdx';
+import JwtPayloadViewer from '../../../components/JwtPayloadViewer';
+
+export default Layout;
+
+## JWT/Token examples
+
+The Dutch iDIN is provided by the Dutch banks through the Dutch Payments Association. The user experience will vary depending on the type of authentication requested.
+
+Four different authentication options are available:
+
+- [Full identification](/verify/e-ids/dutch-idin/#full-identification)
+- [Re-identification](/verify/e-ids/dutch-idin/#re-identification)
+- [Safe login](/verify/e-ids/dutch-idin/#safe-login)
+- [Age verification](/verify/e-ids/dutch-idin/#age-verification)
+
+Different options will render user profiles with different JWT claims.
+
+Developers can specify the user profile per authorize request by providing the `login_hint` parameter with the required value. Alternatively, the value can be supplied in the `scope` parameter.
```

Comment:
I'd assume that `scope=idin:re-id` is the preferred way, the `login_hint` is mostly for intermediaries that does not pass through scopes?

Reply:
thank you, will update